src/CtaLink/CtaLink.js

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ctaLinkIcons, ctaLinkDisplay, ctaLinkColors, ctaLinkAnimations, ctaLinkButtonSizes, ctaLinkButtonVariants } from './CtaLink.levers';
+import Icon from 'react-hero-icon';
+import clsxd from 'clsx-dedupe';
+
+/**
+ * CTA Link Component
+ *
+ */
+export const CtaLink = ({ classes = {}, text, srText, display, link, color, isButton, variant, size, icon, animate, ref, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const levers = {};
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  // display
+  if (display && ctaLinkDisplay.includes(display)) {
+    switch (display) {
+      case 'inline-block':
+        levers.display = 'su-inline-block';
+        break;
+
+      case 'block':
+        levers.display = 'su-block';
+        break;
+
+      case 'flex':
+        levers.display = 'su-flex';
+        break;
+    }
+  }
+
+  // color
+  if (color && ctaLinkColors.includes(color) && !isButton) {
+    switch (color) {
+      case 'red':
+        levers.color = 'su-text-digital-red hocus:su-text-black';
+        break;
+
+      case 'blue':
+        levers.color = 'su-text-digital-blue hocus:su-text-black';
+        break;
+
+      case 'white':
+        levers.color = 'su-text-white hocus:su-text-white';
+        break;
+    }
+  }
+
+  // isButton
+  if (isButton) {
+    levers.isButton = 'su-cta-btn su-font-regular su-leading-none';
```

Comment:
This lever name is a bit misleading. Perhaps change it to `levers.wrapper`

Reply:
levers.wrapper sounds good

Reply:
Done